src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -4,27 +4,41 @@ import com.typesafe.scalalogging.LazyLogging
 import com.google.api.services.genomics.model.Operation
 import org.joda.time.DateTime
 import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
 
 object OperationMetadata extends LazyLogging {
+
+  private def dateStringToDateTime(dateTime: Option[AnyRef]): Try[DateTime] = {
+    val tryValue: Try[String] = dateTime map { _.asInstanceOf[String] } match {
+      case Some(x) => Success(x)
+      case None => Failure(new NoSuchFieldException())
+    }
+
+    tryValue flatMap { _.toDate }
+  }
+
   def apply(operation: Operation): OperationMetadata = {
     val metadata = operation.getMetadata.asScala
+    val startedTime = metadata.get("startTime")
+    val endTime = metadata.get("endTime")
+
     new OperationMetadata(metadata("createTime").asInstanceOf[String].toDate,
-      metadata.get("startTime").map(_.asInstanceOf[String].toDate),
-      metadata.get("endTime").map(_.asInstanceOf[String].toDate))
+      startedTime.isDefined,
+      dateStringToDateTime(startedTime),
+      endTime.isDefined,
+      dateStringToDateTime(endTime))
   }
 
-  val defaultDate = DateTime.parse("0")
-
   implicit class EnhancedString(val string: String) extends AnyVal {
-    def toDate: DateTime = try {
+    def toDate: Try[DateTime] = try {
 
-      DateTime.parse(string.asInstanceOf[String])
+      Success(DateTime.parse(string.asInstanceOf[String]))
     } catch {
       case e: Exception =>
-        logger.error(s"Unexpected date string: '$string'.")
-        defaultDate
+        logger.warn(s"Unexpected date string: '$string'.")
+        Failure(e)
```

Comment:
I changed this because failing to parse a value for a log message felt more like something to warn about that an actual error in the workflow.